src/java.base/share/classes/sun/security/ssl/X509KeyManagerImpl.java

```diff
@@ -77,15 +79,58 @@ final class X509KeyManagerImpl extends X509ExtendedKeyManager
     // cached entries
     private final Map<String,Reference<PrivateKeyEntry>> entryCacheMap;
 
+    private final boolean ksP12;
```

Comment:
Can we improve this name? Something like "cachePrivateKeys" would better describe the purpose